config/site_settings.yml

```
@@ -947,6 +947,8 @@ posting:
     max: 36500
   enable_page_publishing:
     default: false
+  show_public_pages_with_login_required:
```

```suggestion
  show_published_pages_login_required:
```
A minor change but I think it should have "published" in there and we can omit things like "with"